x/gov/keeper/keeper_test.go

```diff
@@ -41,6 +41,11 @@ func (suite *KeeperTestSuite) SetupTest() {
 	err = app.BankKeeper.SendCoinsFromModuleToModule(ctx, minttypes.ModuleName, types.ModuleName, coins)
 	suite.NoError(err)
 
+	// Create a new denom 'atom' so the proposal to update fee denom can be tested
+	atomCoins := sdk.NewCoins(sdk.NewCoin("atom", sdk.NewInt(100000)))
+	err = app.BankKeeper.MintCoins(ctx, minttypes.ModuleName, atomCoins)
+	suite.NoError(err)
```

Comment:
It probably would have been better to pick something other than "atom" here since there's a high probability of other things using that denom too.